test/csuite/time_shift_test.sh

```diff
@@ -42,10 +42,18 @@ then
     exit $EXIT_FAILURE
 fi
 
+# Find test_rwlock file from the wiredtiger directory(two level up ../../) level.
+rwlockfile=`find ../../ -type f -name "test_rwlock" -print -quit`
```

Comment:
I think it's better to use ```git rev-parse --show-toplevel``` to go to the wiredtiger home directory.

Reply:
And then just use relative path to find the ```test_rwlock``` file instead of using find.

Reply:
Updated the script to use relative path.